src/KyckglobalServiceProvider.php

```diff
@@ -24,6 +24,7 @@ public function register()
      */
     public function boot()
     {
+        $this->loadMigrationsFrom(realpath(__DIR__.'/../database/migrations'));
```

Comment:
I think this will cause a conflict with the migration scripts that's already inside of the project. I think u should use the following code instead:
```
$this->publishes([
        __DIR__.'/../database/migrations/' => database_path('migrations')
    ], 'kyck-migrations');
```

Reply:
Added and resolved